go.mod

```diff
@@ -1,20 +1,25 @@
-module github.com/kannon92/kueue-operator
+module github.com/openshift/kueue-operator
 
 go 1.23.0
 
+toolchain go1.23.4
+
+replace (
+	github.com/openshift/client-go => github.com/bertinatto/client-go v0.0.0-20241209194446-f30a3ddc774e
+	github.com/openshift/library-go => github.com/dusk125/library-go v0.0.0-20241212055402-9dbaddb63ab9
+)
+
```

Comment:
can we have a TODO or something to remember to fix thiis

Reply:
Fixed in the latest commit.